Data Structure/Linked List/LinkedListHomework1.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+class Node {
+    /** @var int */
+    private $data;
+    /** @var Node */ 
+    private $next; 
+
+    /**
+    * Constructor Node class
+    */
+    public function __construct($data = 0, $next = null)  // default value of $data is 0, $next is null
+    {
+        $this->data = $data; // initial $data
+        $this->next = $next;  // initial $next
+    }
+
+    /**
+    * get data
+    * @return int
+    */
+    public function getData()
+    {
+        return $this->data;
+    }
+
+    /**
+    * set data
+    * @param int $data
+    */
+    public function setData($data)
+    {
+        $this->data = $data;
+    }
+
+    /**
+    * get next
+    * @return Node
+    */
+    public function getNext()
+    {
+        return $this->next;
+    }
+
+    /**
+    * set next
+    * @param Node $next
+    */
+    public function setNext($next)
+    {
+        $this->next = $next;
+    }
+    
+}
+
+class LinkedList { 
+    /** @var Node  head node */
+    private $head;
+
+    /**
+    *@param int $data
+    */
+    public function insert($data)
+    {
+        $newNode = new Node($data); // create a Node
+        if ($this->head == null) {
```

Comment:
use `===` instead of `==` ... however, this case use `is_null` is better
`if (is_null($this->head)) {`